src/AdvancedVI.jl

```diff
@@ -130,8 +130,28 @@ function estimate_objective end
 
 export estimate_objective
 
+# Oejectives
+
+"""
+    subsample(model, batch)
+
+Subsample `model` to use only the datapoints designated by the iterable collection `batch`.
+
+# Arguments
+- `model`: Model subject to subsampling. Could be the target model or the variational approximation.
+- `batch`: Iterable collection of datapoints or indices corresponding to the subsampled "batch."
+
+# Returns 
+- `sub`: Subsampled model.
+"""
+subsample(model::Any, ::Any) = model
```

Comment:
Do I understand correctly that subsampling is a more general operation than just a VI thing? If that's the case, could this be moved to DynamicPPL, or even AbstractPPL?

Also, I wonder if an empty function without methods would make more sense. Is returning the unmodified original model a reasonable fallback? I could imagine it confusing users, who would call it and get a return value, not realising it's actually just the original model.

Reply:
@yebai Any comments on the current direction on the PPL side?